src/client/pages/index/module.vue

```diff
@@ -135,7 +135,7 @@ getHot().then((hot) => {
           </button>
         </div>
         <div border="b main" />
-        <template v-for="tr of filteredTransforms" :key="tr.name">
+        <template v-for="tr of filteredTransforms" :key="`${tr.name}-${tr.index}`">
```

Comment:
Can it be simplified as 
```suggestion
        <template v-for="tr of filteredTransforms" :key="tr.index">
```